tox.ini

```diff
@@ -1,10 +1,10 @@
 [tox]
 envlist =
+    # Django 1.7 does not support Python 3.5+
     py{27,33,34}-django17
-    py{27,33,34,35}-django18
-    py{27,34,35}-django19
-#    py{27,34,35}-django110
-#    py{27,34,35,36}-django111
+    # Django 1.9+ does not support Python 3.3
```

Comment:
You don't need to add those comments. We would also have to mention Python 3.2, which [Django 1.7](https://pypi.python.org/pypi/Django/1.7.11) would support but is excluded from the build matrix from sane reasons.

Please keep a continuous list of Djangos, from `django17` down to `django111` with the Python versions specified with expansion syntax. Thanks!